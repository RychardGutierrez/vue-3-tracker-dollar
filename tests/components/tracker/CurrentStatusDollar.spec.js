const { shallowMount } = require('@vue/test-utils');

import CurrenStatusDollar from '../../../src/components/tracker/CurrentStatusDollar.vue';

describe('CurrentStatusDollar component', () => {
  test('should to match with snapshot', () => {
    const props = {
      converterDollar: {
        result: '6.98',
      },
      simbolMoney: 'BO',
    };

    const wrapper = shallowMount(CurrenStatusDollar, { props });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
