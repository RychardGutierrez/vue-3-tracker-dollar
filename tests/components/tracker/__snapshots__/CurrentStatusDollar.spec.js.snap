// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentStatusDollar component should to match with snapshot 1`] = `
"<div class=\\"flex max-w-xl w-full py-4 px-12 bg-dollar-primary text-white justify-center rounded-md shadow-md mb-12\\">
  <h1 class=\\"text-4xl m-4\\"> 1 USD = 6.98 BO</h1>
</div>"
`;
